cmsranking/static/TimeView.js

```diff
@@ -45,29 +45,19 @@ var TimeView = new function () {
         }, 1000);
         self.on_timer();
 
-        $("#TimeView_selector_elapsed").click(function () {
-            self.status = 0;
-            self.on_timer();
-            $("#TimeView_selector").removeClass("open");
-        });
-
-        $("#TimeView_selector_remaining").click(function () {
-            self.status = 1;
-            self.on_timer();
-            $("#TimeView_selector").removeClass("open");
-        });
+        var bindStatusTo = function(status, id) {
+            $(id).click(function() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["$" is not defined.](https://www.codacy.com/app/s-maggiolo/cms/file/2192941803/issues/source?bid=3167167&fileBranchId=3167185#l49)